python_modules/dagster/dagster/_core/definitions/run_status_sensor_definition.py

```diff
@@ -92,6 +93,9 @@
     Callable[..., RawSensorEvaluationFunctionReturn],
 ]
 
+RUN_STATUS_SENSOR_FETCH_LIMIT = int(os.getenv("DAGSTER_RUN_STATUS_SENSOR_FETCH_LIMIT", "25"))
+RUN_STATUS_SENSOR_PROCESS_LIMIT = int(os.getenv("DAGSTER_RUN_STATUS_SENSOR_PROCESS_LIMIT", "5"))
```

Comment:
env vars vs arguments to this sensor definition - what makes more sense (im not sure)

Reply:
I was also on the fence about this. My latest thinking is that in an ideal world this would not be something the user has to think about at all, so I am hesitant about making them part of the formal sensor API in perpetuity. Ideally we would just hoist up the right filters so that no overfetching is needed, or even maybe figure out what values make sense here based on historical results for the sensor?